docs/wiki/deployment/configuration.md

```diff
@@ -568,6 +568,8 @@ Taking the `tcc_system_entries` ATC table as an example, which controls which pe
 
 `$ sqlite3 /Library/Application\ Support/com.apple.TCC/TCC.db`
 
+Note that if you receive an error on the above, TCC.db is a protected file and requires Full Disk Access.  You may be required to add this to `Terminal` (or `iTerm`, or any other terminal emulator) in order to allow `sqlite3` and `osqueryi` to open this file.
```

Comment:
I agree on principle, the thing is that this table is _really helpful_ for an example for folks doing corporate security which makes it a fantastic ATC example.  

I showed this to a co-worker and they instantly wanted to begin using OSQuery on as many MacOS endpoints as possible.

Are there any examples of note/warning blocks elsewhere in the docs?

Reply:
(As a side note I think this is extra fun because enabling it means that terminal/iterm actually shows up in the query with kTCCServiceSystemPolicyAllFiles)

Reply:
Though are some compelling reasons. Maybe we can put a caveat here, about how they might want to disable FDA after the exercise? I'm not sure, what do you think makes the most sense?